ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/ConflictCheckWithCommittedImpl.scala

```diff
@@ -149,4 +158,96 @@ private[validate] class ConflictCheckWithCommittedImpl(
     }
   }
 
+  private def sameContractData(
```

Comment:
I am wondering why changes in this class did not trigger any change in `ConflictCheckWithCommittedSpec` ?